### PR TITLE
osutil/disks: partition UUID lookup

### PR DESCRIPTION
**Pull request by bboozzoo**
Lookup partition's UUID.

### CODE REVIEW

File: osutil/disks/mockdisk.go [hunk 1]
```diff
@@ -93,6 +93,21 @@ func (d *MockDiskMapping) FindMatchingPartitionWithPartLabel(label string) (Part
 	}
 }
 
+func (d *MockDiskMapping) FindMatchingPartitionWithNode(node string) (Partition, error) {
```
Comment (stolowski) [inline osutil/disks/mockdisk.go:96]: Is this needed by this PR?

**Review by stolowski — APPROVED** [review]
LGTM, one question about FindMatchingPartiionWithNode.

File: osutil/disks/mockdisk.go (hunk 1 above)
Comment (bboozzoo) [reply osutil/disks/mockdisk.go:96]: uhh no, must have missed that when stitching the patches

Comment (bboozzoo) [reply osutil/disks/mockdisk.go]: I've force pushed a change dropping this.

**Pull request merged**